R/bar_plot_xctr.R

```diff
@@ -0,0 +1,56 @@
+#' Create a horizontal stacked bar chart without financial data
+#'
+#' Generate a horizontal stacked bar chart showing the distribution of
+#' the emission risk profiles risks for one or several benchmarks
+#'
+#' @param data A data frame like [without_financial].
+#' @param benchmarks A character vector specifying the benchmarks for which the
+#' emission profiles will be plotted. The user can choose from one to several
+#' benchmark(s) to be plotted.
+#'
+#' @return A [ggplot] object.
+#'
+#' @export
+#'
+#' @examples
+#' benchmarks <- c("all", "unit", "isic_sec")
+#' bar_plot_xctr(without_financial, benchmarks)
+bar_plot_xctr <- function(data,
+                          benchmarks = c(
+                            "all",
+                            "unit",
+                            "tilt_sec",
+                            "unit_tilt_sec",
+                            "isic_sec",
+                            "unit_isic_sec"
+                          )) {
+  benchmarks_arg <- arg_match(benchmarks, multiple = TRUE)
+  # TODO: do we want to drop NA's everywhere silently?
+  data <- data |>
```

Comment:
It'd be best to warn the user that we're dropping rows with missing values -- and remove the comment.